onnxruntime/core/providers/cpu/cpu_execution_provider.cc

```diff
@@ -456,6 +466,7 @@ class ONNX_OPERATOR_VERSIONED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDoma
 class ONNX_OPERATOR_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 11, Conv);
 #ifdef MLAS_F16VEC_INTRINSICS_SUPPORTED
```

Comment:
>MLAS_F16VEC_INTRINSICS_SUPPORTED [](http://example.com/codeflow?start=7&length=32)

nit: put all those fp16 declaration together.

Reply:
They are originally grouped by version. I thought that makes upgrading versions easier. So I tried to preserve this sequence. What do you think?